css-four-column-layout/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Four Column Layout</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Barlow:wght@800&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Barlow&display=swap" rel="stylesheet">
+</head>
+<body>
+
+  <div class="container">
+
+    <div class="row">
+      <div class="column-full">
+        <div class="top">
+          <h2>How It Works</h2>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+
+      <div class="column-quarter">
+        <div class="middle">
+          <img class="top-image" src="./images/screen-1.png" alt="App Switching">
+        </div>
+      </div>
+
+      <div class="column-quarter">
+        <div class="middle">
+          <img class="top-image" src="./images/screen-2.png" alt="Trip Filtering">
+        </div>
+      </div>
+
+      <div class="column-quarter">
+        <div class="middle">
+          <img class="top-image" src="./images/screen-3.png" alt="Service Filters">
+        </div>
+      </div>
+
+      <div class="column-quarter">
+        <div class="middle">
+          <img class="top-image" src="./images/screen-4.png" alt="Stats + Mileage">
+        </div>
+      </div>
+
+    </div>
+
+
+
+    <div class="row">
+
+      <div class="column-quarter">
+        <section>
+          <img class="bottom-image" src="./images/switch-icon.png" alt="switch-icon">
+          <h3>App Switching</h3>
+          <p>Toggle all your apps on and off.</p>
+        </section>
+      </div>
+
+      <div class="column-quarter">
+        <section>
+          <img class="bottom-image" src="./images/filter-icon.png" alt="filer-icon">
+          <h3>Trip Filtering</h3>
+          <p>Evaluates your trip requests.</p>
+        </section>
+      </div>
+
+      <div class="column-quarter">
+        <section>
+          <img class="bottom-image" src="./images/slider-icon.png" alt="slider-icon">
+          <h3>Service Filters</h3>
+          <p>Accepts only your ideal trips.</p>
+        </section>
+      </div>
+
+      <div class="column-quarter">
+        <section>
+          <img class="bottom-image" src="./images/bars-icon.png" alt="bars-icon">
+          <h3>Stats + Mileage</h3>
+          <p>Tracks your trips across all apps.</p>
+        </section>
+      </div>
+    </div>
+
+  </div>
```

Comment:
I can understand why you would separate this into three rows based on setting it up with layout classes, but now your HTML makes it seem like the images, headers, and subtext are not related, and they absolutely are. You should instead have the images, header, and subtext together inside one `column-quarter`

Reply:
![image](https://user-images.githubusercontent.com/117887912/209485396-427f73ea-663d-463a-8372-1a3169038375.png)